.github/workflows/publish_docs.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     build:
         runs-on: ubuntu-latest
         env:
-            MDBOOK_ENV: 0.4.44
```

Comment:
The [error](https://github.com/mozilla-services/autopush-rs/actions/runs/15982630130/job/45080290720#step:5:16) cites that the preprocessor is running 0.4.36. I'm not sure where that preprocessor is or what container it might be in. Is it possible to isolate that and figure out the dependency or update the dependency in the pre-processor container?